extensions/kubectl-cli/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "kubectl-cli-tool",
+  "displayName": "kubectl CLI Tool for Podman Desktop",
+  "description": "Install and update kubectl CLI Tools without leaving Podman Desktop",
+  "version": "0.0.1",
+  "icon": "icon.png",
+  "publisher": "redhat",
+  "license": "Apache-2.0",
+  "engines": {
+    "podman-desktop": "^0.0.1"
+  },
+  "main": "./dist/extension.js",
+  "contributes": {
+    "cliTools": {
+      "name": "kubectl",
+      "displayName": "kubectl",
+      "markdownDescription": "A command line tool used to run commands against Kubernetes clusters. It does this by authenticating with the Master Node of your cluster and making API calls to do a variety of management actions. If you're just getting started with Kubernetes, prepare to be spending a lot of time with kubectl",
```

Comment:
"Control Plane"

https://kubernetes.io/blog/2022/04/07/upcoming-changes-in-kubernetes-1-24/

https://github.com/kubernetes/enhancements/tree/master/keps/sig-cluster-lifecycle/kubeadm/2067-rename-master-label-taint

Reply:
Updated text and added link to additional info similar to compose.

Reply:
yes I would avoid `master` here